bin/scrape_software_versions.py

```diff
@@ -4,47 +4,50 @@
 import re
 
 regexes = {
-    'nf-core/diaproteomics': ['v_pipeline.txt', r"(\S+)"],
-    'Nextflow': ['v_nextflow.txt', r"(\S+)"],
-    'OpenMS': ['v_openms.txt', r"Version: (\S+)"],
-    'Pyprophet': ['v_pyprophet.txt', r"version: (\S+)"],
-    #'Msproteomicstools': ['v_msproteomicstools.txt', r"(\S+)"]
+    "nf-core/diaproteomics": ["v_pipeline.txt", r"(\S+)"],
+    "Nextflow": ["v_nextflow.txt", r"(\S+)"],
+    "FastQC": ["v_fastqc.txt", r"FastQC v(\S+)"],
```

Comment:
Check the versions here, probably due to template update the OpenMS and Pyroprophet versions are gone

Reply:
done